test/testdata/deployednettemplates/recipes/scenario1/Makefile

```diff
@@ -1,12 +1,16 @@
 PARAMS=-w 100 -R 8 -N 20 -n 100 -H 10 --node-template node.json --relay-template relay.json --non-participating-node-template nonPartNode.json
+FILEPARAMS=--rounds 5000 --ntrx 1000 --naccounts 3000000 --nassets 20000 --napps 20000 --wallet-name "wallet1"
```

Comment:
for now, avoid modifying the `scenario1`, since it's being used as a reference recipe. Instead, you can create a copy of that recipe.